static/app/components/events/interfaces/performance/spanEvidence.tsx

```diff
@@ -14,14 +14,18 @@ import {
 import DataSection from '../../eventTagsAndScreenshot/dataSection';
 import KeyValueList from '../keyValueList';
 import TraceView from '../spans/traceView';
-import {RawSpanType, SpanEntry} from '../spans/types';
+import {RawSpanType, SpanEntry, TraceContextType} from '../spans/types';
 import WaterfallModel from '../spans/waterfallModel';
 
 interface Props {
   event: EventTransaction;
   organization: Organization;
 }
 
+export type TraceContextSpanProxy = Omit<TraceContextType, 'span_id'> & {
+  span_id: string;
+};
+
```

Comment:
🤔 is this type here because `trace?.span_id` gets mad about `span_id` maybe being `undefined`? Is it possible to type narrow by checking `typeof trace.context.trace.span_id` in the conditional below?